internal_event_handlers.go

```diff
@@ -39,6 +39,8 @@ import (
 var _ workflowEnvironment = (*workflowEnvironmentImpl)(nil)
 var _ workflowExecutionEventHandler = (*workflowExecutionEventHandlerImpl)(nil)
 
+const queryTypeStackTrace string = "__stack_trace"
```

Comment:
I think we can make constants for stack trace (and other built in queries) public

Reply:
will change to public and move to Client.go